experimental/sdk/y-azure-webpubsub/README.md

```diff
@@ -1,15 +1,81 @@
-# y-azure-webpubsub
+# Yjs binding for Azure WebPubSub
 
-> Azure Web PubSub Service binding for Yjs
+## Introduction
 
-## Quickstart
+- [Microsoft Azure Web PubSub](https://docs.microsoft.com/en-us/azure/azure-web-pubsub/overview) is a real-time messaging cloud service.
 
-### Install dependencies
+- [Yjs](https://github.com/yjs/yjs) is a [CRDT implementation](https://github.com/yjs/yjs#Yjs-CRDT-Algorithm) that exposes its internal data structure as shared types. Shared types are common data types like Map or Array with superpowers: changes are automatically distributed to other peers and merged without merge conflicts.
 
-```bash
-npm i y-azure-webpubsub
-```
+This package implements a classical client-server model, which helps developers use Microsoft Azure WebPub service to distribute changes without conflicts.
 
-### Licience
+## Usage
 
-[The MIT License](./LICENSE) © Microsoft
\ No newline at end of file
+### Server
+
+1. Install required packages.
+
+    ```bash
+    npm install @azure/web-pubsub
+    npm install y-azure-webpubsub
+    ```
+
+1. Create a host connection to handle conflicts and distribute changes.
+
+    > Note that there is only 1 host connection is allowed for each topic.
+
+    ```ts
+    import { WebPubSubServiceClient } from "@azure/web-pubsub";
+    import { AzureWebPubSubConnection } from "y-azure-webpubsub";
+
+    const client: WebPubSubServiceClient = new WebPubSubServiceClient(
+      connectionString ?? "",
+      "sync"

```

Comment:
must it be "sync"? if it is a requirement, maybe wrap the servieClient into the Connection?

Reply:
No, hub name can be anything, I would like to change it into "<hub>" instead.